server/back-end/models/wordmodel.js

```diff
@@ -0,0 +1,52 @@
+const sql = require("./db.js");
+const { response } = require("express");
+
+// Word
+const Word = function (word) {
+    this.word_name = word.name;
+    this.created_at = Date.now()// add new timestamp ;
+    this.updated_at = Date.now()// add new timestamp ;
+    this.use_yn = word.use_yn;
+    this.word_icon = word.word_icon;
+    this.category_idx = word.category_idx;
```

Comment:
According to this, is there table for categories as well?

Reply:
Yes

Reply:
Got it, can you/nalin complete that by the end of the week or is more time needed?